src/main/java/org/betonquest/betonquest/objectives/BlockObjective.java

```diff
@@ -28,34 +32,58 @@ public class BlockObjective extends CountingObjective implements Listener {
 
     private final boolean noSafety;
 
+    private final CompoundLocation location;
```

Comment:
Please do not introduce new Qodana warnings if possible

Reply:
Could you enlighten me on what you mean by that and how to resolve it?

Reply:
Qodana is another testing tool that runs here in GitHub, you can see it if you click on the diff here in the PR.

It complains that there is no javadoc on the class fields that you introduced.